src/main/scala/org/scalajs/cli/Scalajsld.scala

```diff
@@ -81,15 +100,24 @@ object Scalajsld {
         .text("Execute the specified main(Array[String]) method on startup")
       opt[File]('o', "output")
         .valueName("<file>")
-        .required()
-        .action { (x, c) => c.copy(output = x) }
-        .text("Output file of linker (required)")
+        .action { (x, c) => c.copy(output = Some(x)) }
+        .text("Output file of linker (deprecated)")
+      opt[File]("outputDir")
```

Comment:
I'm not a fan of not having a short option for a de-facto required flag (it's only not required for deprecated usage).